operations/iuf/examples/iuf_list_stages.md

```diff
@@ -0,0 +1,28 @@
+# `iuf list-stages` Examples
+
+(`ncn-m001#`) List the stages for activity `joe-install-20230107-2` along with their status and duration values.
+
+```bash
+iuf -a joe-install-20230107-2 list-stages
++----------------------------+--------------------------------------------------------------------------------------------+-----------+----------+
+| stage                      | description                                                                                | status    | duration |
++----------------------------+--------------------------------------------------------------------------------------------+-----------+----------+
+| process-media              | Inventory and extract products in the media directory for use in subsequent stages         | Succeeded | 0:01:30  |
+| pre-install-check          | Perform pre-install readyness checks                                                       | Succeeded | 0:00:22  |
```

Comment:
Change readyness to readiness

Reply:
Will be addressed when we update to a newer example of `iuf` output.  These typos were in the code itself and have since been fixed.

Reply:
fixed